test/publicApiSpec.js

```diff
@@ -299,6 +430,18 @@ describe('Public API method tests', function () {
                 expect(dtp.daysOfWeekDisabled).toBeDefined();
             });
         });
+
+        describe('access', function () {
+            xit('gets days of week disabled', function () {
```

Comment:
There appears to be a separate issue with this function in that it doesn't work unless you explicitly set it to an array before hand, so I've raised #1459 to track this.